frontend/apps/reader/readerui.lua

```diff
@@ -747,4 +747,12 @@ function ReaderUI:switchDocument(new_file)
     self:showReader(new_file)
 end
 
+function ReaderUI:getCurrentPage()
```

Comment:
Was about to suggest to put it as a more appropriate place (that at the end), with some other alike functions... but there are none :)
So, indeed, it's not the module where we expect such functions... so let it be there.
May be just add a comment `-- little convenient helper`
(Or move if into ReaderView, where it would be more at home?)